frame/migrations/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "pallet-migrations"
+version = "1.0.0"
+description = "FRAME pallet to execute multi-block migrations."
+authors = ["Parity Technologies <admin@parity.io>"]
+homepage = "https://substrate.io"
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/paritytech/substrate"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [ "derive"] }
+scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
+
+frame-benchmarking = { version = "4.0.0-dev", default-features = false, optional = true, path = "../benchmarking" }
+frame-support = { version = "4.0.0-dev", default-features = false, path = "../support" }
+frame-system = { version = "4.0.0-dev", default-features = false, path = "../system" }
+impl-trait-for-tuples = "0.2.2"
+log = "0.4.18"
+sp-std = { version = "8.0.0", path = "../../primitives/std", default-features = false }
+sp-runtime = { version = "24.0.0", path = "../../primitives/runtime", default-features = false }
+docify = "0.1.14"
+
+[dev-dependencies]
+sp-tracing = { version = "10.0.0", path = "../../primitives/tracing", features = [ "std" ] }
+sp-core = { version = "21.0.0", path = "../../primitives/core", features = [ "std" ] }
+sp-io = { version = "23.0.0", path = "../../primitives/io", features = [ "std" ] }
+pretty_assertions = "1.3.0"
+docify = "0.1.14"
+
+[features]
+default = ["std"]
+
+std = ["codec/std", "frame-benchmarking?/std", "frame-support/std", "frame-system/std", "scale-info/std", "sp-std/std", "sp-runtime/std"]
```

Comment:
interesting style :D

Reply:
Will reformat. A CI that checks the ordering and style would be nice.